tests/TextTest.php

```diff
@@ -7,36 +7,54 @@
 use OCP\Files\Folder;
 
 class TextTest extends \PHPUnit\Framework\TestCase {
+	private static $attachmentNames = [
+		'aaa.png',
+		'aaa (2).png',
+		'aaa 2).png',
+		'aaa (2.png',
+		'aaa ((2.png',
+		'aaa 2)).png',
+		'a[a]a.png',
+		'a(a)a.png',
+		'a](a.png',
+		',;:!?.§-_a_',
+		'a`a`.png',
+	];
+
 	public function testDummy() {
 		$app = new Application();
 		$this->assertEquals('text', $app::APP_NAME);
 	}
 
-	public function testGetAttachmentNamesFromContent() {
-		$contentNames = [
-			'aaa.png',
-			'aaa (2).png',
-			'aaa 2).png',
-			'aaa (2.png',
-			'aaa ((2.png',
-			'aaa 2)).png',
-			'a[a]a.png',
-			'a(a)a.png',
-			'a](a.png',
-			',;:!?.§-_a_',
-			'a`a`.png',
-		];
+	public function testGetOldAttachmentNamesFromContent() {
```

Comment:
Something for a separate PR but we should move those tests into tests/unit/Service/ImageServiceTest.php